src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,95 @@
+    },
+    "Parameters": {
+        "Username": "",
+        "DiskFilter": "SizeGreaterThan:1000gb",
+        "LinuxCudaVersion": "12.0",
+        "LinuxDriverVersion": "525",
+        "LinuxLocalRunFile": "https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run"
+    },
+    "Actions": [
+        {
+            "Type": "MLPerfTrainingExecutor",
+            "Parameters": {
+                "Scenario": "training-mlperf-bert-batchsize-45-gpu-8",
+                "Model": "bert",
+                "Username": "$.Parameters.Username",
+                "DiskFilter": "$.Parameters.DiskFilter",
+                "BatchSize": "45",
+                "Implementation": "pytorch-22.09",
+                "ContainerName": "language_model",
+                "DataPath": "mlperf-training-data-bert.1.0.0",
+                "GPUNum": "8",
+                "ConfigFile": "config_DGXA100_1x8x56x1.sh"
+            }
+        }
+    ],
+    "Dependencies": [
+        {
+            "Type": "FormatDisks",
+            "Parameters": {
+                "Scenario": "InitializeDisks"
+            }
+        },
+        {
+            "Type": "MountDisks",
+            "Parameters": {
+                "Scenario": "MountDataDisk",
+                "MountPointName": "/mlperftraining",
+                "DiskFilter":  "BiggestSize"
+            }
+        },
+        {
+            "Type": "GitRepoClone",
+            "Parameters": {
+                "Scenario": "CloneMLCommonsRepo",
+                "RepoUri": "https://github.com/mlcommons/training_results_v2.1.git",
+                "PackageName": "mlperf"
+            }
+        },
+        {
+            "Type": "NvidiaCudaInstallation",
+            "Parameters": {
+                "Scenario": "InstallNvidiaCuda",
+                "LinuxCudaVersion": "$.Parameters.LinuxCudaVersion",
+                "LinuxDriverVersion": "$.Parameters.LinuxDriverVersion",
+                "Username": "$.Parameters.Username",
+                "LinuxLocalRunFile": "$.Parameters.LinuxLocalRunFile"
+            }
+        },
+        {
+            "Type": "DockerInstallation",
+            "Parameters": {
+                "Scenario": "InstallDocker"
+            }
+        },
+        {
+            "Type": "NvidiaContainerToolkitInstallation",
+            "Parameters": {
+                "Scenario": "InstallNvidiaContainerToolkit"
+            }
+        },
+        {
+            "Type": "DCGMIInstallation",
+            "Parameters": {
+                "Scenario": "InstallDCGMI"
+            }
+        },
+        {
+            "Type": "DependencyPackageInstallation",
+            "Parameters": {
+                "Scenario": "InstallMLPerfTrainingDataPackage",
+                "BlobContainer": "packages",
+                "BlobName": "mlperf-training-data-bert.1.0.0.zip",
+                "PackageName": "MLPerfTrainingData_bert",
```

Comment:
we are keeping the package name all lowercase, so less likely to make user mistake

Reply:
Updated.